airflow/dags/mental_health_dag.py

```diff
@@ -0,0 +1,129 @@
+import json
+import logging
+
+import pendulum
+import pymsteams
+import requests
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+from airflow.models import Variable
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
```

Comment:
please view how logging is set up in Airflow: it's done with a special configuration parameter, not like this